### PR TITLE
Upgrade to Grails 6.x 

### PR DESCRIPTION
**Pull request by vsachinv**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,106 +1,185 @@
...
     maven { url "https://repo.grails.org/grails/repo" }
+    maven { url "https://repo.grails.org/grails/core" }
+    maven { url "https://build.shibboleth.net/nexus/content/repositories/releases" }
+    maven { url "https://build.shibboleth.net/nexus/content/groups/public/" }
+    maven { url "https://code.lds.org/nexus/content/groups/main-repo" }
+    maven { url "https://repository.jboss.org/maven2/" }
+    maven { url "http://jasperreports.sourceforge.net/maven2"
+        allowInsecureProtocol = true
+    }
+    maven { url "http://10.100.3.6:8080/nexus/content/repositories/common-component-snapshots"
```
Comment (jeetmp3) [inline build.gradle]: Is this a private URL? If yes then please remove it

File: build.gradle [hunk 2]
```diff
@@ -1,106 +1,185 @@
...
+        allowInsecureProtocol = true
+    }
+    maven { url "http://10.100.3.6:8080/nexus/content/repositories/thirdparty"
```
Comment (jeetmp3) [inline build.gradle]: same URL mentioned here

Comment (vsachinv) [reply build.gradle:31]: Yes will remove the same.

File: build.gradle (hunk 1 above)
Comment (vsachinv) [reply build.gradle:28]: Yes will remove the same.

**Comment by vsachinv** [comment]
Closing the PR. Will raise a new PR.

**Comment by vsachinv** [comment]
\

**Comment by vsachinv** [comment]
@jeetmp3 Could you please delete your comment and delete this PR? if possible.

**Comment by jeetmp3** [comment]
@vsachinv I don't think this is allowed by Github. Let me know if you find anything

**Comment by vsachinv** [comment]
@jeetmp3 see if you could delete review comments ? I will delete mine.

**Comment by vsachinv** [comment]
@jeetmp3  Please see https://docs.github.com/en/communities/moderating-comments-and-conversations/managing-disruptive-comments for comment deleting.

**Pull request closed without merging**